src/renderer/components/composer/EmojiAndStickerPicker.tsx

```diff
@@ -68,22 +70,46 @@ export const StickerPicker = (props: {
     runtime.openPath(folder)
   }
 
+  const stickerPackNames = Object.keys(stickers)
+
   return (
     <div className='sticker-picker'>
-      <button onClick={onOpenStickerFolder}>Open Sticker Folder</button>
-      <div className='sticker-container'>
-        {Object.keys(stickers).map(stickerPackName => {
-          return (
-            <StickerDiv
-              chatId={chatId}
-              key={stickerPackName}
-              stickerPackName={stickerPackName}
-              stickerPackImages={stickers[stickerPackName]}
-              setShowEmojiPicker={setShowEmojiPicker}
-            />
-          )
-        })}
-      </div>
+      {stickerPackNames.length > 0 ? (
+        <>
+          <div className='sticker-container'>
+            {stickerPackNames.map(name => (
+              <DisplayedStickerPack
+                chatId={chatId}
+                key={name}
+                stickerPackName={name}
+                stickerPackImages={stickers[name]}
+                setShowEmojiPicker={setShowEmojiPicker}
+              />
+            ))}
+          </div>
+          <div className='sticker-actions-container'>
+            <button
+              className='delta-button-round'
+              onClick={onOpenStickerFolder}
+            >
+              {tx('open_sticker_folder')}
```

Comment:
this button is too bold if there is already content.

the look of a secondary button is totally sufficient here (otherwise it looks as if it needs to be clicked to proceed)

(secondary button = transparent background, border in button color, if at all)

Reply:
addressed via https://github.com/deltachat/deltachat-desktop/pull/3293/commits/11443b40e1ac7dabf039e04f5229f683165a249e

lmk if you think the border should also be removed? currently just uses the existing style definition for the secondary button